tests/python/unit/test_spacy_tokenizer.py

```diff
@@ -3,10 +3,14 @@
 # pytest automatically runs all the function starting with "test_"
 # see https://docs.pytest.org for more information
 
+import os
+
 import pandas as pd
 
 from spacy_tokenizer import MultilingualTokenizer  # noqa
 
+stopwords_folder_path = os.getenv("STOPWORDS_FOLDER_PATH")
```

Comment:
Handle the case where there is no stopwords as an env variable, check if the file exists and log ?

Reply:
I want to test precisely that the library works with `stopwords_folder_path`

Reply:
To me, you should have at least one isolated test to assess that the files your plugin relies on exist, a failing test on that would quickly point out the failure

Reply:
I see. At the moment the code won't fail if the files don't exist. I should change that and then the test will check for failure. Not sure that an isolated test is needed to check for the files, the library will take care of that.

Reply:
Should be done by db39689, I let you re-check